var/spack/repos/builtin/packages/py-ipython-cluster-helper/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyIpythonClusterHelper(PythonPackage):
+    """Quickly and easily parallelize Python functions using IPython on
+    a cluster, supporting multiple schedulers. Optimizes IPython defaults
+    to handle larger clusters and simultaneous processes.
+    """
+
+    homepage = "https://github.com/roryk/ipython-cluster-helper"
+    url      = "https://github.com/roryk/ipython-cluster-helper/archive/v0.6.4.tar.gz"
+
+    version('0.6.4', sha256='6c8b42e1428305eeb0c54d1a8ecf3c551ec9ee02e1f039b4b12260bef66fc446')
+    version('0.6.3', sha256='0256e6f72c31f846fd3bf7ed0a87bc362d77a0731cb52dbdc19c41686e44faeb')
+
+    depends_on('python', type=('build', 'run'))
+    depends_on('py-setuptools@18.5:', type='build')
```

Comment:
Does this package install any files in `prefix.bin` that import `pkg_resources`? If so then this will need to be type=build/run

Reply:
Fixed.